static/docs/user-guide/dvcignore.md

```diff
@@ -38,7 +38,107 @@ by design works similar way as `.gitignore` does.
 
 The same as for [`.gitignore`](https://git-scm.com/docs/gitignore).
 
-## Example
+## Examples: Modification of ignored data
+
+Lets see if what happens when we modify ignored file.
```

Comment:
`if what` - typo

Reply:
Also `Lets`->`Let's`. I fixed both in c8591d4